### ISSUE TITLE
The namespace "jaw" is ambiguous

### ISSUE THREAD

**Issue by tebaly** [jawira/doctrine-diagram-bundle#11]
Symphony console allows command shortcuts. But

```
$ bin/console jaw:doc
  The namespace "jaw" is ambiguous.  
  Did you mean one of these?         
      jawira                         
      jawira:doctrine.  
```

Wrong hint. These commands do not exist.

```
$ bin/console jawira
  Command "jawira" is not defined. 
```

```
$ bin/console jawira:doctrine
  Command "jawira:doctrine" is not defined
```

For example

```
$ bin/console doctrine

 Command "doctrine" is not defined.             
                                                 
  Did you mean one of these?                     
      doctrine:cache:clear-collection-region     
      doctrine:cache:clear-entity-region         
      doctrine:cache:clear-metadata              
      doctrine:cache:clear-query                 
      doctrine:cache:clear-query-region          
      doctrine:cache:clear-result                
      doctrine:database:create                   
      doctrine:database:drop     
    < ... >
```

**Comment by jawira** [comment]
Hi, thanks for reporting this, currently I don't have time to work on this, but it will be fixed when possible.

**Comment by jawira** [comment]
Hello this is fixed in `v0.6`, please note command name has changed to `doctrine:diagram`, so now you have to use `bin/console do:di` for example.

![image](https://github.com/jawira/doctrine-diagram-bundle/assets/496541/ec0720b9-5383-4da1-9ef4-4dd4773dacfa)

Also option names have changed.
